src/routes/[id]/training/queues/[queueId]/+page.server.ts

```diff
@@ -0,0 +1,25 @@
+import prisma from '$lib/prisma';
+import type { PageServerLoad } from './$types';
+
+export const load: PageServerLoad = async ({ params, depends }) => {
+    depends("queue:data");
+
+    console.log("Running load function...");
```

Comment:
Probably can remove this print statement

Reply:
Good catch, have removed!